app/components/preprint-status-banner/component.js

```diff
@@ -57,7 +57,7 @@ export default Ember.Component.extend({
             CLASS_NAMES[this.get('submission.reviewsState')];
     }),
 
-    didReceiveAttrs() {
+    didInsertElement() {
```

Comment:
`didInsertElement` is generally used for things that mess with the DOM, which doesn't seem to be the case here.